locales/zh_CN/LC_MESSAGES/gui.po

```diff
@@ -305,39 +326,48 @@ msgid ""
 "gui.get_event(ti.GUI.PRESS)\n"
 "\n"
 "# if ESC pressed or SPACE released:\n"
-"gui.get_event((ti.GUI.PRESS, ti.GUI.ESCAPE), (ti.GUI.RELEASE, "
-"ti.GUI.SPACE))"
+"gui.get_event((ti.GUI.PRESS, ti.GUI.ESCAPE), (ti.GUI.RELEASE, ti.GUI.SPACE))"
 msgstr ""
+"# 如果按下或释放ESC:\n"
+"gui.get_event(ti.GUI.ESCAPE)\n"
+"\n"
+"# 如果按下任何键:\n"
+"gui.get_event(ti.GUI.PRESS)\n"
+"\n"
+"# 如果按ESC或释放SPACE:\n"
+"gui.get_event((ti.GUI.PRESS, ti.GUI.ESCAPE), (ti.GUI.RELEASE, ti.GUI.SPACE))"
 
 #: ../../gui.rst:187 ../../gui.rst:212 ../../gui.rst:230 ../../gui.rst:250
 msgid "(GUI)"
-msgstr ""
+msgstr "(GUI)"
 
 #: ../../gui.rst:188 ../../gui.rst:213
 msgid "(optional, EventFilter) filter out matched events"
-msgstr ""
+msgstr "（可选，EventFilter）过滤掉匹配的事件"
```

Comment:
```suggestion
msgstr "（可选，事件过滤器）过滤掉匹配的事件"
```
Yeah, it's a class, but since you already translated that L318 and didn't add a claim like `事件过滤器(event filter)`, let's translate this too.